### PR TITLE
Relay Chain - Coretime Overview UI Tab

### PR DESCRIPTION
**Pull request by piggydoughnut**
## Relay chain

> NOTE: 17th October is the end of the 1st cycle for Polkadot, and I want to test see how the table behaves at that moment, so for now the PR is in draft

This PR separates Coretime Chain - Overview UI Tab from this https://github.com/polkadot-js/apps/pull/10383

Original Feature Request : https://github.com/polkadot-js/apps/issues/10285

Relay chain does not provide enough information to show, so we are using Coretime Endpoints to get extra information in order to provide better user experience.

Changes:
- Exposing Coretime endpoint so it can be used from any chain now, just like the People endpoint

Parachain table: 
- Renewal and Renewal price are only shown if there is a record for the given chain in the `potentialRenewals` calls from `broker` pallet.

Unrelated changes:
- Updated names for Paseo networks to start with Paseo, as we filter by it at some occasions

Screenshot 2024-10-16 at 19 52 31

## Coretime Chain

Coretime chain Overview Tab also got small updates. 

- added colourful labels
- new summary items: cycle dates and cycle progress

Screenshot 2024-10-16 at 19 54 52

Closes #10383

### CODE REVIEW

**Comment by TarikGul** [comment]
looks really great so far!

**Comment by piggydoughnut** [comment]
@TarikGul it is ready for you 😊  Thank you.

File: packages/page-broker/src/utils.ts [hunk 1]
```diff
@@ -61,15 +60,26 @@ export const estimateTime = (targetTimeslice: string | number, latestBlock: numb
...
+ *
+ * @param data: CoreWorkloadType[]
+ * @param core: core number
+ * @param currentRegion
+ * @param timeslice
+ * @param param4
+ * @param regionLength
+ * @returns
+ */
+
```
Comment (TarikGul) [inline packages/page-broker/src/utils.ts:73]:
```suggestion
```

**Review by TarikGul — APPROVED** [review]
Amazing!

**Pull request merged**